pkgs/by-name/ba/bandcamp-dl/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  fetchPypi,
+  fetchFromGitHub,
+  python313Packages,
+}:
+
+let
+  unicode-slugify =
+    let
+      pname = "unicode-slugify";
+      version = "0.1.5";
+    in
+    python313Packages.buildPythonPackage {
+      format = "setuptools";
+      inherit pname version;
+
+      src = fetchPypi {
+        inherit pname version;
+        sha256 = "sha256-JfQkJYMX5MtBCT4pUzdLOvHyMJcpdmRzHNs65G9r1sM=";
+      };
+
+      propagatedBuildInputs = with python313Packages; [
+        six
+        unidecode
+      ];
+
+      doCheck = false;
+    };
+in
```

Comment:
this should probably be its own package in python-modules with tests+metadata.

also, use the original (github, gitlab, etc) source instead of pypi when possible